reframe/core/modules.py

```diff
@@ -449,8 +412,7 @@ def _module_command_failed(self, completed):
         return re.search(r'ERROR', completed.stderr) is not None
 
     def _exec_module_command(self, *args, msg=None):
```

Comment:
I am not very happy with this. I could not find any better suitable design. Do you have any idea how to improve this?